src/config/mod.rs

```diff
@@ -234,6 +234,43 @@ impl PartialConfig {
     }
 }
 
+fn check_semver_version(range_requirement: &str, actual: &str) -> bool {
+    let mut version_req = match semver::VersionReq::parse(range_requirement) {
+        Ok(r) => r,
+        Err(e) => {
+            eprintln!("Error: failed to parse required version: {}", e);
+            return false;
+        }
+    };
+    let actual_version = match semver::Version::parse(actual) {
+        Ok(v) => v,
+        Err(e) => {
+            eprintln!("Error: failed to parse current version: {}", e);
```

Comment:
Let's include the `actual` version in the error message:

```suggestion
            eprintln!("Error: failed to parse current version {actual:?}: {e}");
```